### PR TITLE
Update Capability Statement References

### PR DESCRIPTION
**Pull request by emichaud998**
Pull requests into this repository require the following checks to be completed by the submitter and reviewers.

Changes in this PR:
Changed all capability statement references to include conformance, and fixed dashboard sidebar to not have text spillover with new longer tab names.

**Submitter:**
- [x] Administrative tasks are complete.
  - This pull request describes why these changes were made.
  - The JIRA ticket links to this PR.
  - The JIRA ticket for this PR is: [Lantern 366 JIRA Ticket](https://oncprojectracking.healthit.gov/support/browse/LANTERN-366)
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.
- [x] Code review has been performed.
  - Code diff has been done and checked to ensure only expected code is being committed.
  - A linter for the language has been run.

**Primary Reviewer:**

- [x] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [x] Code review has been performed.
  - Code is maintainable, reusable, efficient, and correct.
  - Code accomplishes the tasks purpose.
  - Code follows style guidance appropriate for the language, including passing linter checks.
  - Code is well commented (which does not mean verbosely commented).
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.

**Secondary Reviewer:**

- [ ] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [ ] Code review has been performed.
  - Code accomplishes the tasks purpose.
- [ ] Tests are complete.
  - Tests have been run locally and pass.

**Merger:**

Prior to merging, ensure that all checks have been completed. Delete the branch after the merge
is complete.

Create a new branch `<branch name>_gomodupdate`. Run `make update_mods` to update the `go.mod`
and `go.sum` files to point to master. Create a PR. Require one sanity check reviewer.

### CODE REVIEW

File: shinydashboard/lantern/functions/endpoints.R [hunk 1]
```diff
@@ -404,7 +404,7 @@ get_endpoint_security_counts <- function(db_connection) {
     "Total Indexed Endpoints", as.integer(isolate(app_data$fhir_endpoint_totals()$all_endpoints)),
     "Endpoints with successful response (HTTP 200)", as.integer(isolate(app_data$response_tally()$http_200)),
     "Endpoints with unsuccessful response", as.integer(isolate(app_data$response_tally()$http_non200)),
-    "Endpoints without valid capability statement", as.integer(get_no_cap_statement_count(db_connection)),
+    "Endpoints without valid capability statement / conformance statement", as.integer(get_no_cap_statement_count(db_connection)),
```
Comment (bmath10) [inline shinydashboard/lantern/functions/endpoints.R:407]: Based on Jeff's comment, it seems like anywhere where we had "capability statement" it should be CapabilityStatement

Comment (bmath10) [reply shinydashboard/lantern/functions/endpoints.R:407]: Also "conformance statement" should just be "Conformance"

Comment (bmath10) [reply shinydashboard/lantern/functions/endpoints.R:407]: It also seems like here and basically everywhere else we should be referring to them as "resources", so this should read "Endpoints without valid CapabilityStatement/Conformance Resource"

Comment (bmath10) [reply shinydashboard/lantern/functions/endpoints.R:407]: So I'd apply this here and all of the changes below!

**Review by bmath10 — CHANGES_REQUESTED** [review]

File: shinydashboard/lantern/functions/endpoints.R (hunk 1 above)
Comment (emichaud998) [reply shinydashboard/lantern/functions/endpoints.R]: Yeah this makes much more sense, thank you for clarifying! Just made all these changes throughout all the files!

File: shinydashboard/lantern/modules/capabilitystatementsizemodule.R [hunk 2]
```diff
@@ -13,7 +13,7 @@ capabilitystatementsize_UI <- function(id) {
         htmlOutput(ns("notes_text"))
       ),
         column(width = 4,
-        h4("Capability Statement Size Statistics"),
+        h4("CapabilityStatement / Conformance Size Statistics"),
```
[the remark this answers is not in the document]
Comment (emichaud998) [reply shinydashboard/lantern/modules/capabilitystatementsizemodule.R]: Sounds good, I think that sounds better too!

**Review by bmath10 — APPROVED** [review]
